fix(vpc): add limitations and faq (#4479)
scaleway/docs-content: faq/vpc.mdx

productIcon: VpcProductIcon
---

## VPC basics

### What is the difference between VPC and a Private Network?

One default VPC (**V**irtual **P**rivate **C**loud) for every available region is automatically created in each Scaleway [Project](/organizations-and-projects/concepts/#project). A VPC offers layer 3 network isolation.

Within each VPC, you can create multiple **Private Networks** and attach Scaleway resources to them, as long as the resources are in an AZ within the network's region. Attached resources can then communicate between themselves in an isolated and secure layer 2 network, away from the public internet.

In the future, VPC will allow you to interconnect your VPC with other networks, define access control lists and more.

### What happened to my classic, mono-AZ Private Network?

When VPC and regional Private Networks moved from Public Beta to General Availability, all mono-AZ Private Networks were automatically migrated to be regional. [Read the documentation](/vpc/reference-content/vpc-migration/) to find out more about the migration process.

> ### What is a default VPC and why can't I delete it?

Reviewer: ```suggestion
### What is a default VPC and why can not I delete it?
```

Scaleway currently has three regions: Paris, Amsterdam and Warsaw. One default VPC is automatically created for each region, in each Scaleway [Project](/organizations-and-projects/concepts/#project). Any new Private Networks that you create will be added to the default VPC for their region, unless you override this by specifying a different VPC.

You cannot delete a default VPC, but you can rename it, and/or create other VPCs and use those rather than the default VPCs, if you prefer. Default VPCs do not prevent you from deleting an otherwise empty Project.

### How much does it cost to create a VPC, Private Network or reserved private IP addresses?

The following resources and features are free of charge:

- VPCs and VPC routing
- Private Networks (except for [Elastic Metal servers](https://www.scaleway.com/en/pricing/elastic-metal/) and [Apple silicon](https://www.scaleway.com/en/pricing/apple-silicon/))
- Reserved private IP addresses on IPAM

> ### Why can't I delete my Private Network even though it's empty?

Reviewer: ```suggestion
### Why can I not delete my Private Network even though it is empty?
```

You might have a reserved IP address that is blocking the deletion - check out our [troubleshooting page](/vpc/troubleshooting/cant-delete-vpc-pn/).

## VPC routing

### Can I route traffic between different Private Networks on the same VPC?

Yes, [VPC routing](/vpc/concepts#routing) allows you to automize the routing of traffic between resources in different Private Networks within the same VPC.

### Can I route traffic between different Private Networks in different VPCs or different Scaleway Projects?

This is not currently possible. You may consider using a VPN tunnel to achieve this, for example [IPsec](https://en.wikipedia.org/wiki/IPsec) or [WireGuard](https://en.wikipedia.org/wiki/WireGuard). Scaleway also offers an [OpenVPN InstantApp](/tutorials/openvpn-instant-app/), making it easy to install a VPN directly on an Instance.

> ### Why can't I route traffic to my Managed Database on another Private Network?

Reviewer: ```suggestion
### Why can not I route traffic to my Managed Database on another Private Network?
```
